src/components/popup/WalletHeader.tsx

```diff
@@ -526,13 +526,13 @@ const AppAction = styled.div`
   display: flex;
 `;
 
-export const Action = styled(CopyIcon)<{ active?: boolean }>`
+export const Action = styled(CopyIcon)<{ $active?: boolean }>`
```

Comment:
We should use [transient props](https://styled-components.com/docs/faqs#transient-props-since-51) to avoid propagating undesired props/attributes to the HTML nodes (which will show up as an error on the Console).